src/style.css

```
@@ -0,0 +1,107 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: 800;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, serif;
+}
+
+.search-bar {
+  position: relative;
+  margin-top: 20px;
+  box-sizing: border-box;
+
+  --color-of-border-input: #e1e7ed;
+  --color-of-input-background: #fff;
+}
+
+.search-bar__input {
+  border: 1px solid var(--color-of-border-input);
+  background: var(--color-of-input-background);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+  width: 100%;
+  flex: 1;
+  outline: none;
+}
+
+.search-bar__input--big {
+  line-height: 68px;
+  padding: 0 0 0 62px;
+}
+
+.search-bar__input--big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-bar__input--big:focus {
+  background: linear-gradient(
+    180deg, 
+    var(--color-of-input-background) 0%,
+    #f6f6f7 100%
+  );
+  border-radius: 3px;
+  font-family: Avenir-Heavy, serif;
+  font-size: 16px;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
```

into `.search-bar__input`.
```suggestion
  background: linear-gradient(
    180deg, 
    var(--color-of-input-background) 0%,
    #f6f6f7 100%
  );
  border-radius: 3px;
  font-family: Avenir-Heavy, serif;
  font-size: 16px;
  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
```